smdebug/tensorflow/base_hook.py

```diff
@@ -218,6 +219,14 @@ def export_collections(self):
         collection_file_name = f"{self.worker}_collections.json"
         self.collection_manager.export(self.out_dir, collection_file_name)
 
+    def has_default_hook_configuration(self):
```

Comment:
why this function is repeated again here?

Reply:
I have overridden the fn in core hook to utilize a constant defined for TF.